netsuite/10.0/modules/ROOT/pages/netsuite-reference.adoc

```diff
@@ -1422,7 +1422,7 @@ This operation returns the result of a job given that it has finished processing
 |===
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | | x
-| Job Id a| String |  The ID of the job. |  `#[payload]` |
+| Job Id a| String |  The ID of the job. |  |
 | Page Index a| Number |  The page number of the asynchronous result. |  1 |
 | Target Variable a| String |  The name of a variable to store the operation's output. |  |
```

Comment:
```suggestion
| Target Variable a| String |  Name of a variable in which to store the operation's output |  |
```
I think that this is the wording we need instead of "variable to store" throughout.